### PR TITLE
Symfonize attendance hours' edition and save

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

**Review by alexarn — CHANGES_REQUESTED** [review]
The link for adding working is wrong (attendance/add => workinghour/add).
In the same URL, the parameter retour is "index.php". Shouldn't it be 'workinghour ?'

Please rebuild this PR on master. Back links are on the former index page

**Review by alexarn — CHANGES_REQUESTED** [review]
Link on my account is the former one

**Comment by sillydebs** [comment]
Main issues were :
- wrong link in monCompte.php 
- wrong parameters or variables for "retour" or href in template

Corrections are done and I force-pushed due to the rebase I did 2 hours ago.

**Review by alexarn — CHANGES_REQUESTED** [review]
- Back link is wrong. Please test. click on add workinghours, click on back (retour) => fail

- from my account, the link to working hours is denied (workingHour)

- Total hours are not calculated at the fly

- i can save workinghours for the same agent on the period. We should have an ajax control

- when i create new exception, the table is not pre-filled with the current hours. It's empty

**Comment by sillydebs** [comment]
Many files have been corrupted after my attempt to restore the branch. I will fix it ASAP.

**Review by alexarn — CHANGES_REQUESTED** [review]
AttendanceController and attendance/edit.html.twig are not needed imo

File: public/monCompte.php [hunk 1]
```diff
@@ -94,7 +94,7 @@
     if ($config['PlanningHebdo-Agents']) {
         echo <<<EOD
     <div style='display: inline-block; width:300px; position: absolute; right: 22px; text-align: right; margin-top:22px;'>
-    <a href='index.php?page=planningHebdo/modif.php&retour=monCompte.php' class='ui-button'>
+    <a href='/workinghour/add?retour=monCompte.php' class='ui-button'>
```
Comment (alexarn) [inline public/monCompte.php:97]: Please add $config['URL']

File: public/monCompte.php [hunk 2]
```diff
@@ -141,7 +141,7 @@
         $arrow=$elem['remplace']?"<font style='font-size:20pt;'>&rdsh;</font>":null;
 
         echo "<tr>";
-        echo "<td style='white-space:nowrap;'>$arrow <a href='index.php?page=planningHebdo/modif.php&amp;id={$elem['id']}&amp;retour=monCompte.php'/>";
+        echo "<td style='white-space:nowrap;'>$arrow <a href='/workinghour/{$elem['id']}?retour=monCompte.php'/>";
```
Comment (alexarn) [inline public/monCompte.php:144]: Please add $config['URL']

File: public/planningHebdo/js/script.planningHebdo.js [hunk 3]
```diff
@@ -242,23 +242,23 @@ function plHebdoMemePlanning(){
...
   if(confirm("Etes vous sûr(e) de vouloir supprimer ce planning de présence ?")){
-    
+
     var CSRFToken = $('#CSRFSession').val();
     // Suppression du planning en arrière plan
-    
+
     $.ajax({
-      url: "planningHebdo/ajax.delete.php",
+      url: "/workinghour",
```
Comment (alexarn) [inline public/planningHebdo/js/script.planningHebdo.js:250]: Please add baseURL

**Review by alexarn — CHANGES_REQUESTED** [review]
Missing atomic update for removing access

Critical issue: sites are not kept in the working hours form

**Review by alexarn — CHANGES_REQUESTED** [review]
"Parse error: syntax error, unexpected 'echo' (T_ECHO), expecting ';' or ',' in /home/plb/planningBiblio/public/monCompte.php on line 97" when going to my account

File: public/setup/atomicupdate/removePaths.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$sql[]="DELETE FROM `{$dbprefix}acces` WHERE `page` LIKE '%planningHebdo%'";
```
Comment (alexarn) [inline public/setup/atomicupdate/removePaths.php:3]: You need to remove the route /workinghour

**Review by alexarn — CHANGES_REQUESTED** [review]
Other issue:
  - Set PlanningHebdo-Validation-N2 to 1
  - Add new workinghour for an agent
  - Set status to "Accepted (waiting for ....)"
  - Save
  - Edit the workinghour
    => No option accepted in status

**Review by alexarn — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**